src/com/codisimus/plugins/phatloots/PhatLootsAPI.java

```diff
@@ -115,6 +120,26 @@ public static boolean loot(Block block, Player player, boolean autoSpill) {
             return false;
         }
 
+        World wgWorld = BukkitAdapter.adapt(block.getLocation().getWorld());
+        ApplicableRegionSet applicableRegionSet = WorldGuard.getInstance().getPlatform().getRegionContainer().get(wgWorld).getApplicableRegions(BukkitAdapter.asBlockVector(block.getLocation()));

```

Comment:
These changes should not be in PhatLootsAPI and instead should be in a separate PluginHookManager specifically for WorldEdit. This code will error on servers that do not have WorldEdit installed.

Reply:
I did some changes, moved to WorldGuardManager. Are they convenient ?